EFCore.Seeder.Console/Program.cs

```diff
@@ -9,7 +9,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            var csvConfiguration = new CsvHelper.Configuration.Configuration
+            var csvConfiguration = new CsvHelper.Configuration.CsvConfiguration(new System.Globalization.CultureInfo("en-US")) // .Configuration
```

Comment:
Can you replace this for the following lines? It's a better practice to define the culture in a single place that's used throughout the code (even when it's not used anywhere else at the moment).

`CultureInfo.CurrentCulture = new CultureInfo("en-US");`

`var csvConfiguration = new CsvHelper.Configuration.CsvConfiguration(CultureInfo.CurrentCulture)`

Reply:
I agree with you. Will do that and respond after changes are made and tested.

Reply:
Hi @DanielAltamirano, I have made recommended changes. All Test ran successfully.
Please take a look. 
Thank you.